### ISSUE TITLE
Advanced tooltips

### ISSUE THREAD

**Issue by KebiainWoods** [almende/vis#952]
Some kind of delay or close option on the tooltips, to allow the mouse to enter the tip and click links.

**Comment by AlexDM0** [comment]
Hi, 

This seems similar to #944. You could also just create your own popup for this on the popup event. All the information you need for this is available in the event and through the methods.

Regards,

Alex

**Comment by KebiainWoods** [comment]
Would you be interest in being paid to get this working for us? I know I’ll struggle to get this working quickly. I’m a designer, I need to get back to making things look pretty :)


On 9 Jun 2015, at 16:55, Alex <notifications@github.com> wrote:

> Hi,
> 
> This seems similar to #944. You could also just create your own popup for this on the popup event. All the information you need for this is available in the event and through the methods.
> 
> Regards,
> 
> Alex
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Hi Kevin,

It's really not that much work so I wont charge for it :).

Here you go: just make it look pretty :)
http://jsbin.com/neladaqimi/5/

If you feel so inclined to return the favor, we're looking for someone to style the timeline original style into something more modern ;)

Regards,

Alex

**Comment by KebiainWoods** [comment]
That’s very good of you thanks.

Happy to do some work on the timeline. What’s the best way for me to go about that? Do you have some files I can download and work on locally?

thanks


On 10 Jun 2015, at 09:56, Alex <notifications@github.com> wrote:

> Hi Kevin,
> 
> It's really not that much work so I wont charge for it :).
> 
> Here you go: just make it look pretty :)
> http://jsbin.com/neladaqimi/5/
> 
> If you feel so inclined to return the favor, we're looking for someone to style the timeline original style into something more modern ;)
> 
> Regards,
> 
> Alex
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Hi Kevin,

you can download them here: 
http://visjs.org/#download_install

we think the css classes of the items and just general styling could be improved, modernized. 

You can run the examples locally and play with the css, the code does not require changes for this. Take a look at this example: http://visjs.org/examples/timeline/styling/customCss.html

Themes could also be awesome, so you could go any direction :). 

Regards

**Comment by AlexDM0** [comment]
I noticed that the hidePopup event is a little buggy. I'll fix that in the next release.

Regards

**Comment by AlexDM0** [comment]
Here is the workaround and some code that the popup does not disappear when the mouse is over it.
http://jsbin.com/neladaqimi/13/edit?html,output

**Comment by KebiainWoods** [comment]
That’s perfect for what we need thanks. I’ll take a look at styling the timeline this afternoon.




On 10 Jun 2015, at 10:59, Alex <notifications@github.com> wrote:

> Here is the workaround and some code that the popup does not disappear when the mouse is over it.
> http://jsbin.com/neladaqimi/13/edit?html,output
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by KebiainWoods** [comment]
How about something like this?





On 10 Jun 2015, at 10:27, Alex <notifications@github.com> wrote:

> Hi Kevin,
> 
> you can download them here: 
> http://visjs.org/#download_install
> 
> we think the css classes of the items and just general styling could be improved, modernized.
> 
> You can run the examples locally and play with the css, the code does not require changes for this. Take a look at this example: http://visjs.org/examples/timeline/styling/customCss.html
> 
> Themes could also be awesome, so you could go any direction :).
> 
> Regards
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Hi Kevin, 

I think you didnt add a link somewhere?

Regards

**Comment by KebiainWoods** [comment]
Was there no image attached? I’ve mocked up in photoshop first. Helps me when designing to not have my css hat on.


On 10 Jun 2015, at 14:33, Alex <notifications@github.com> wrote:

> Hi Kevin,
> 
> I think you didnt add a link somewhere?
> 
> Regards
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Hi Kevin,

I cant see an image, you can also email it to alex@almende.org

Regards

**Comment by KebiainWoods** [comment]
You can see it here…  http://dev73.xyidigital.com/timeline.jpg

This is the page we’re building with your help by the way…   http://dev73.xyidigital.com/bees.html

There seems to be a bit of flickering on the rollovers at the moment, maybe something need a tweak?



On 10 Jun 2015, at 16:02, Alex <notifications@github.com> wrote:

> Hi Kevin,
> 
> I cant see an image, you can also email it to alex@almende.org
> 
> Regards
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
The mockup looks good :) 

I'll take a look at the flickering tomorrow. 

Regards

**Comment by KebiainWoods** [comment]
Ok great. I’ll start working it up into the html page.

thanks.


On 10 Jun 2015, at 16:10, Alex <notifications@github.com> wrote:

> The mockup looks good :) 
> 
> I'll take a look at the flickering tomorrow. 
> 
> Regards 
> 
> 
> 
> > On 10 jun. 2015, at 17:07, KebiainWoods <notifications@github.com> wrote:
> > 
> > You can see it here… http://dev73.xyidigital.com/timeline.jpg
> > 
> > This is the page we’re building with your help by the way… http://dev73.xyidigital.com/bees.html
> > 
> > There seems to be a bit of flickering on the rollovers at the moment, maybe something need a tweak?
> > 
> > 
> > 
> > On 10 Jun 2015, at 16:02, Alex <notifications@github.com> wrote:
> > 
> > > Hi Kevin,
> > > 
> > > I cant see an image, you can also email it to alex@almende.org
> > > 
> > > Regards
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub.
> > > 
> > 
> > Kevin Woods
> > 07720716792
> > XYi Digital Solutions Ltd
> > www.xyidigital.com
> > 
> > 
> > 
> > —
> > Reply to this email directly or view it on GitHub.
> > 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Here's a fix for the flickering ( we didnt clear the timeout when a new one was loaded ) 

http://jsbin.com/semuxozome/2/edit?html,output

**Comment by AlexDM0** [comment]
I think this issue can be closed, as for the timeline design, if you get around to it you can email it to me or make a new issue. I'd prefer not to go too far off track in this issue.

Regards and thanks :)

**Comment by AlexDM0** [comment]
Hi Kevin,

Any luck with the css? I look forward to your results.

Regards

**Comment by KebiainWoods** [comment]
Hi Alex,

I send you a file yesterday (around 17.30) to your other email address. Could you check to see if you have it.

Thanks
Kevin


On 16 Jun 2015, at 15:14, Alex <notifications@github.com> wrote:

> Hi Kevin,
> 
> Any luck with the css? I look forward to your results.
> 
> Regards
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Hi Kevin,

It seems to have been put in my spam folder. I've slapped my email around so it should be doing better now :).

Me and Jos have looked at the file you've sent and it looks good! We're not too sure about the red though, maybe the green you had in your mockup would be better? On selecting I think the green from your mockup looks much better than the outline (maybe paired with the blue line)? The number boxes (in the timeAxis) are a bit tall but that could also just need some getting used to it. Big things are modern after all :).

We really like the dark/light of the timeAxis and the background! The green is my personal favorite as the color factor. The blue could be nice too though.

Anyway, great work and I look forward to any refinements you want to make! :+1: 

Just to be safe, if you email me again, let me know here as well. I'm not sure how obedient my email is after just one punishment.

Regards,

Alex

**Comment by josdejong** [comment]
Thanks Kevin! I like your design. Way better than the current style :) I like the red color, but our concern was whether the red lines attract too much attention, at the expense of the item contents, which should attract most attention.

**Comment by KebiainWoods** [comment]
Ok, will do. I’ll get back to this and fine tune as soon as I get a chance this week.

Thanks


On 16 Jun 2015, at 15:35, Alex <notifications@github.com> wrote:

> Hi Kevin,
> 
> It seems to have been put in my spam folder. I've slapped my email around so it should be doing better now :).
> 
> Me and Jos have looked at the file you've sent and it looks good! We're not too sure about the red though, maybe the green you had in your mockup would be better? On selecting I think the green from your mockup looks much better than the outline (maybe paired with the blue line)? The number boxes (in the timeAxis) are a bit tall but that could also just need some getting used to it. Big things are modern after all :).
> 
> We really like the dark/light of the timeAxis and the background! The green is my personal favorite as the color factor. The blue could be nice too though.
> 
> Anyway, great work and I look forward to any refinements you want to make! 
> 
> Just to be safe, if you email me again, let me know here as well. I'm not sure how obedient my email is after just one punishment.
> 
> Regards,
> 
> Alex
> 
> —
> Reply to this email directly or view it on GitHub.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com

**Comment by AlexDM0** [comment]
Hi Kevin,

I was wondering if you've had any time to look at the second iteration :) In case I missed another email please let me know! 

We'd like to make your theme the new default theme once everything is ready and provide the legacy one as an extra css.

Regards,

Alex

**Comment by KebiainWoods** [comment]
Hi Alex

I’m having all kinds of trouble with my email at the moment. Can you let me know if you receive this file this time please.

Thanks
Kevin








> On 9 Jul 2015, at 14:50, Alex <notifications@github.com> wrote:
> 
> Hi Kevin,
> 
> I was wondering if you've had any time to look at the second iteration :) In case I missed another email please let me know!
> 
> We'd like to make your theme the new default theme once everything is ready and provide the legacy one as an extra css.
> 
> Regards,
> 
> Alex
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/almende/vis/issues/952#issuecomment-119975290>.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com <http://www.xyidigital.com/>

**Comment by AlexDM0** [comment]
Hi Kevin, I just saw your two questions. I had to get them from my spambox again. I would suggest you use git for all network related questions so we're sure to get them.

To answer, yes it is sort of possible to make the network responsive, you will have to give the container div  a width and height in percentages. The network adapts to the window size but it doesnt change the camera so if you make the window smaller, the network will not scale accordingly. This is a design choice on our side which we may revert in the near future.

I have not received a file yet though, my spambox may have deleted it... We could also continue the discussion on the design here, maybe thats easier? I'll email you a website you can use for file sharing.

Regards,

Alex

**Comment by AlexDM0** [comment]
Hi Kevin,

I've not seen an email from you. Have you received my email? Sorry if this seems pushy, I want to be sure the communication lines are working :)

Regards

**Comment by KebiainWoods** [comment]
Hello testing testing?????




> On 17 Jul 2015, at 14:05, Alex <notifications@github.com> wrote:
> 
> Hi Kevin,
> 
> I've not seen an email from you. Have you received my email? Sorry if this seems pushy, I want to be sure the communication lines are working :)
> 
> Regards
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/almende/vis/issues/952#issuecomment-122268247>.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com <http://www.xyidigital.com/>

**Comment by AlexDM0** [comment]
Hi Kevin,

I see your git reply :) (to check, only the testing testing one) no emails though..

Regards

**Comment by KebiainWoods** [comment]
Hi, which email address can I send a file directly to? I’ll send from another account.



> On 20 Jul 2015, at 17:45, Alex <notifications@github.com> wrote:
> 
> Hi Kevin,
> 
> I see your git reply :) (to check, only the testing testing one) no emails though..
> 
> Regards
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/almende/vis/issues/952#issuecomment-122945189>.
> 

Kevin Woods
07720716792
XYi Digital Solutions Ltd
www.xyidigital.com <http://www.xyidigital.com/>

**Comment by AlexDM0** [comment]
alex@almende.org is fine. 

Regards,

Alex

**Comment by AlexDM0** [comment]
Hi Kevin, 

I've received your email. We'll get to work on it :) We're not sure in which release we'll put it yet or which color (or white) would be the default item.

Thanks a lot for the effort!

Regards
